basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/NumberOrderingNodeType.java

```diff
@@ -0,0 +1,65 @@
+package club.mondaylunch.gatos.basicnodes;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import club.mondaylunch.gatos.core.data.DataBox;
+import club.mondaylunch.gatos.core.data.DataType;
+import club.mondaylunch.gatos.core.graph.connector.NodeConnector;
+import club.mondaylunch.gatos.core.graph.type.NodeType;
+
+public class NumberOrderingNodeType extends NodeType.Process {
+
+    public static final DataType<Mode> NUMBER_ORDERING_MODE = DataType.register("number_ordering_type", Mode.class);
+
+    @Override
+    public Map<String, DataBox<?>> settings() {
+        return Map.of("mode", NUMBER_ORDERING_MODE.create(Mode.GREATERTHAN));
+    }
+
+    @Override
+    public Set<NodeConnector.Input<?>> inputs(UUID nodeId, Map<String, DataBox<?>> settings) {
+        return Set.of(
+            new NodeConnector.Input<>(nodeId, "inputA", DataType.NUMBER),
+            new NodeConnector.Input<>(nodeId, "inputB", DataType.NUMBER)
```

Comment:
These can probably just be named 'a' and 'b'?

Reply:
Convention with other nodes has been "input/output<letter>". If, for whatever reason, this node had multiple outputs the distinction would be useful to preserve.